src/firefly/js/ui/SimpleComponent.jsx

```diff
@@ -34,3 +34,36 @@ export class SimpleComponent extends PureComponent {
         }
     }
 }
+
+
+
+/**
+ * A replacement for SimpleComponent.
+ * This function make use of useState and useEffect to
+ * trigger a re-render of functional components when the value in the store changes.
+ *
+ * @param stateGetters  one or more functions returning a state
+ * @returns {Object[]}  an array of state's value in the order of the given stateGetters
+ */
+export function useStoreConnector(...stateGetters) {
+
+    const rval = [];
+    const setters = stateGetters.map((getter) => {
+        const [val, setter] = useState(getter());
+        rval.push(val);
+        return [getter, setter];
+    });
+
+    const isMounted = true;
```

Comment:
I think you could remove isMount.  It is never false and I don't think it is need for functional components anyway.

Reply:
I was pretty sure I was getting the setState on unmounted error before when I attempted to do this last time.  Googled and the recommendation was to do this.  But, I will try again.  We can easily add back if the error shows up again.